Core/Core/Kits/KitDeclaration.cs

```diff
@@ -26,39 +27,4 @@ public ISpeckleConverter LoadConverter(string app)
   {
     return null;
   }
-
-  public Base ToSpeckle(object @object)
-  {
-    throw new NotImplementedException();
-  }
-
-  public bool CanConvertToSpeckle(object @object)
-  {
-    throw new NotImplementedException();
-  }
-
-  public object ToNative(Base @object)
-  {
-    throw new NotImplementedException();
-  }
-
-  public bool CanConvertToNative(Base @object)
-  {
-    throw new NotImplementedException();
-  }
-
-  public IEnumerable<string> GetServicedApplications()
-  {
-    throw new NotImplementedException();
-  }
-
-  public void SetContextDocument(object @object)
-  {
-    throw new NotImplementedException();
-  }
-
-  public bool TryLoadConverter(string app, out ISpeckleConverter speckleConverter)
-  {
-    throw new NotImplementedException();
-  }
```

Comment:
I don't understand how this works at all 🥲

Reply:
Yes now that I look at this twice, this seems like at some point we the converter in the `CoreKit`.

Let's not get ahead of ourselves and remove that class, but I'm tempted!

Marking this as resolved.